rdrf/rdrf/models/definition/models.py

```diff
@@ -1499,6 +1499,7 @@ class ContextFormGroup(models.Model):
                                  related_name="context_form_groups",
```

Comment:
`code` needs to be included in the importer & exporter.

Should also look at using it in the importer's `get_or_create` call instead of the `name`.